packages/memory-storage/src/memory-storage.ts

```diff
@@ -120,6 +121,18 @@ export class MemoryStorage implements storage.StorageClient {
         return new RequestQueueClient({ id, baseStorageDirectory: this.requestQueuesDirectory, client: this, ...options });
     }
 
+    setStatusMessage(message: string, options: storage.SetStatusMessageOptions = {}): Promise<void> {
+        s.string.parse(message);
+        s.object({
+            isStatusMessageTerminal: s.boolean.optional,
+        }).parse(options);
+
+        const log = new Log();
+        log.info(`Setting${options.isStatusMessageTerminal ? ' terminal' : ''} status message: ${message}`);
```

Comment:
why not use the default logger instance here? its a bit weird to create new logger instance every time the method gets called